### PR TITLE
Assignment4

### PR DESCRIPTION
**Pull request by AlbertoRodriguezm**

### CODE REVIEW

File: Assignment4/AlbertoRodriguezm-w140323/Task06.java [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+			Individual jane = model.createIndividual(ns + "Jane Smith", model.getOntClass(ns + "Researcher"));
+
+			// ** TASK 6.5: Add to the individual JaneSmith the fullName, given and family
+			// names **
+			jane.addLiteral(VCARD.FN, "Sarah Jane Smith");
+			jane.addLiteral(VCARD.Given, "Jane");
+			jane.addLiteral(VCARD.Family, "Sarah");
+
+			// ** TASK 6.6: Add UPM as the university where John Smith works **
+			model.getIndividual(ns + "JohnSmith").addProperty(model.getProperty(ns + "worksIn"),
```
Comment (dachafra) [inline Assignment4/AlbertoRodriguezm-w140323/Task06.java]: where Jhon Smith is working? Your triple needs an object

File: Assignment4/AlbertoRodriguezm-w140323/Task07.java [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+		}
+		ExtendedIterator subclasses2 = model2.getOntClass(ns+"Person").listSubClasses();
+		while (subclasses2.hasNext())
+		{
+			OntClass subclass2 = (OntClass) subclasses2.next();
+			System.out.println("Subclass of Person: "+subclass2.getURI());
+		}
+	
+	}
+	private static void listarInstaciasSubclases(String uri) {
```
Comment (dachafra) [inline Assignment4/AlbertoRodriguezm-w140323/Task07.java]: you don't need this code

File: Assignment4/AlbertoRodriguezm-w140323/Task06.java (hunk 1 above)
Comment (AlbertoRodriguezm) [reply Assignment4/AlbertoRodriguezm-w140323/Task06.java]: model.getIndividual(ns + "JohnSmith").addProperty(model.getProperty(ns + "worksIn"), model.createIndividual(ns + "UPM", model.getOntClass(ns + "University")));

:JohnSmith :worksIn :UPM

**Comment by AlbertoRodriguezm** [comment]
Done changes

**Comment by dachafra** [comment]
well done!

**Pull request merged**